### PR TITLE
actor(ticdc): add actor log

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: ref #4976

### What is changed and how it works?
add log for actor goroutine lifetime

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: pkg/actor/system.go [hunk 1]
```diff
@@ -447,7 +447,15 @@ func (s *System) Start(ctx context.Context) {
 	for i := 0; i < s.numWorker; i++ {
 		id := i
 		s.wg.Go(func() error {
-			defer pprof.SetGoroutineLabels(ctx)
+			defer func() {
+				log.Info("actor goroutine exited",
+					zap.Int("id", id),
+					zap.String("name", s.name))
+				pprof.SetGoroutineLabels(ctx)
```
Comment (overvenus) [inline pkg/actor/system.go:454]:
```suggestion
```

We don't need to set labels as it is exiting.

File: pkg/actor/system.go [hunk 2]
```diff
@@ -566,6 +574,9 @@ func (s *System) poll(ctx context.Context, id int) {
 			if !running {
 				// Close the actor and mailbox.
 				p.onActorClosed()
+				log.Info("actor is closed",
+					zap.Uint64("id", uint64(p.mb.ID())),
+					zap.String("name", s.name))
```
Comment (overvenus) [inline pkg/actor/system.go:579]: We need to add the log after `p.onMailboxEmpty` too.

Comment (overvenus) [reply pkg/actor/system.go:579]: It may be very verbose, when there are many tables.

Comment (sdojjy) [reply pkg/actor/system.go:578]: removed this log

**Review by 3AceShowHand — APPROVED** [review]

**Review by overvenus — APPROVED** [review]

**Pull request merged**